### PR TITLE
Create tool to generate variation selector cmap.

### PR DESCRIPTION
**Pull request by dougfelt**
Adds data and tool, and updates setup.py so noto-emoji will be able to use it.

### CODE REVIEW

File: third_party/ucd/emoji-data.txt [hunk 1]
```diff
@@ -0,0 +1,1315 @@
+# Emoji Data for UTR #51
```
Comment (roozbehp) [inline third_party/ucd/emoji-data.txt]: You need to update the README.third_party file to point to location and version of this file.

File: nototools/add_vs_cmap.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+VS_TEXT = 0xFE0E
+VS_EMOJI = 0xFE0F
+
+def modify_font(font_name, font, presentation, emoji_variants):
+  cmap_table = font_data.get_cmap(font)
+  emoji = set(cmap_table.keys()) & emoji_variants
+  if not emoji:
+    print 'no emoji match those in %s' % font_name
+    return
+  uvs = VS_EMOJI if presentation == 'emoji' else VS__TEXT
```
Comment (roozbehp) [inline nototools/add_vs_cmap.py]: Why two underscores? (pylint should be able to catch these.)

Comment (dougfelt) [inline nototools/add_vs_cmap.py]: because that's a typo...

**Comment by dougfelt** [comment]
Thanks Roozbeh.

FYI, the protocol James and I have been following is the reviewer LGTMs the
pull request, and then the person who initiated it (who generally has write
permission in all these cases) merges it.

On Wed, Oct 14, 2015 at 4:38 PM, Roozbeh Pournader <notifications@github.com

**Comment by roozbehp** [comment]
Sure. OK.

**Pull request merged**